src/main/scala/org/ergoplatform/modifiers/state/UtxoSnapshot.scala

```diff
@@ -0,0 +1,41 @@
+package org.ergoplatform.modifiers.state
+
+import org.ergoplatform.modifiers.ErgoPersistentModifier
+import org.ergoplatform.modifiers.history.Header
+import org.ergoplatform.settings.Algos
+import scorex.core.ModifierTypeId
+import scorex.core.serialization.Serializer
+import scorex.crypto.authds.ADDigest
+import scorex.util.{ModifierId, bytesToId}
+
+case class UtxoSnapshot(manifest: UtxoSnapshotManifest,
```

Comment:
Please add class comments for `UtxoSnapshot`, `UtxoSnapshotChunk` and `UtxoSnapshotManifest` describing their goals and functions

Reply:
@catena2w, done.